common/strutils.cpp

```diff
@@ -45,7 +45,11 @@ namespace al {
 
 std::optional<std::string> getenv(const char *envname)
 {
+#ifdef _GAMING_XBOX
+    const char *str{::getenv(envname)};
+#else
     const char *str{std::getenv(envname)};
+#endif
```

Comment:
For unknown reason `std::getenv` is not available for non desktop apps:
```
#ifdef _CRT_USE_WINAPI_FAMILY_DESKTOP_APP
using _CSTD getenv;
using _CSTD system;
#endif // _CRT_USE_WINAPI_FAMILY_DESKTOP_APP
```
however `getenv` is available